src/MizarWidgets/include/MizarWidgets/SamplingWithFrameTrackWidget.h

```diff
@@ -58,6 +62,9 @@ class SamplingWithFrameTrackWidget : public QWidget {
       QStringLiteral("Probability of false-alarm for at least one function:");
   static const inline QString kMultiplicityCorrectionDisabledLabel =
       QStringLiteral("Probability of false-alarm for an individual function:");
+
+  static const inline SamplingWithFrameTrackReportConfigValidator kValidator{kBaselineTitle,
```

Comment:
I think the variable name should be more descriptive. At least `kConfigValidator`, or maybe even more.

Reply:
Let's call it `kConfigValidator`. Making it longer is unnecessary, as the config itself is clear out of the context.